batavia/core/version.js

```diff
@@ -8,7 +8,10 @@ magic_map[String.fromCharCode(238, 12, 13, 10)] = [[3, 4], [0xF]]
 magic_map[String.fromCharCode(248, 12, 13, 10)] = [[3, 5], [0xA, 0]]
 magic_map[String.fromCharCode(22, 13, 13, 10)] = [[3, 5], [0xF]]
 magic_map[String.fromCharCode(23, 13, 13, 10)] = [[3, 5, 3], [0xF]]
-magic_map[String.fromCharCode(51, 13, 13, 10)] = [[3, 6, 2], [0xF]]
+magic_map[String.fromCharCode(23, 13, 13, 10)] = [[3, 5], [0xF]]
```

Comment:
This looks suspicious to me. It's overwriting the value from the previous line, isn't it?

Reply:
@martica Thanks, I cleaned this duplicate up and removed all the version that are < 3.5 :+1: